go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/elgris/sqrl v0.0.0-20210727210741-7e0198b30236
 	github.com/lib/pq v1.10.9
 	github.com/pentops/log.go v0.0.0-20230815045424-6ebbd9ef2576
-	github.com/pentops/o5-go v0.0.0-20240103191826-234a4df7c841
+	github.com/pentops/o5-go v0.0.0-20240108181354-69adf8151c27
```

Comment:
Noting here so you also know. I caught and was reminded of this morning, the o5-pb and o5-go repos are not automatically linked together. A change to the pb requires us to do the regen and push of the go code.

Reply:
👍 I pushed the protobufs and its updates to o5-go and verified changes from https://github.com/pentops/o5-pb/pull/15 were correct in Dante.